stdlib/3/urllib/parse.pyi

```diff
@@ -97,9 +97,16 @@ class SplitResultBytes(_SplitResultBytesBase, _NetlocResultMixinBytes): ...
 class ParseResultBytes(_ParseResultBytesBase, _NetlocResultMixinBytes): ...
 
 
+@overload
 def parse_qs(qs: str, keep_blank_values: bool = ..., strict_parsing: bool = ..., encoding: str = ..., errors: str = ...) -> Dict[str, List[str]]: ...
```

Comment:
This seems like a good use case for `AnyStr`: the function takes in str or bytes and returns the matching type.